content/machine-config/_index.md

```diff
@@ -133,3 +133,52 @@ spec:
         path: /etc/sysctl.d/99-elasticsearch.conf
 EOF
 ```
+
+# Modify MTU in a second interface in workers
+
+Sometimes a storage network interface is attached to nodes in order to reach an external storage. Probably you could need to modify the MTU in those interfaces. 
```

Comment:
`Sometimes a storage network interface is attached to nodes in order to reach an external storage. In order to improve the performance, you could need to modify the MTU in those interfaces to 9000 (aka. jumboframes)`

Reply:
Change done, thanks